src/igmpproxy.h

```diff
@@ -35,8 +35,10 @@
 *   igmpproxy.h - Header file for common includes.
 */
 
-#include "config.h"
-#include "os.h"
+#ifndef __FreeBSD__
+    #include "config.h"
```

Comment:
`config.h` must be always included and must be included as the first file. This is required for autoconfig/automake system. Disabling it inclusion on FreeBSD just makes autoconfig/automake broken.

Reply:
https://github.com/pali/igmpproxy/issues/35

You're right about config.h of course, will fix that. 

I thought I would finally include patch for this too.